libs/core/langchain_core/runnables/history.py

```diff
@@ -25,14 +24,14 @@
 )
 
 if TYPE_CHECKING:
+    from langchain_core.chat_history import BaseChatMessageHistory
```

Comment:
could we use langchain_core.chat_history.base?

Reply:
there is no base